cpp/Platform.Delegates/Platform.Delegates.CreateDelegate.h

```diff
@@ -1,29 +1,27 @@
-#pragma once
-
-#ifndef PLATFORM_DELEGATES_CREATEDELEGATE
-#define PLATFORM_DELEGATES_CREATEDELEGATE
+#ifndef PLATFORM_DELEGATES_CREATEDELEGATE_H
+#define PLATFORM_DELEGATES_CREATEDELEGATE_H
 
 #include "Platform.Delegates.Delegate.h"
 
 namespace Platform::Delegates
 {
     template <typename ReturnType, typename... Args>
-    static Delegate<ReturnType(Args...)> CreateDelegate(ReturnType(&function)(Args...))
```

Comment:
Why did you remove static keyword?

Reply:
static specifier means internal linkage, so every compilation unit will have his own version of this functions.

Reply:
Good to know. Looks like `static` keyword have different meanings in different contexts.